### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by virtuald** [jpieper/pygazebo#6]
I intend to adapt pygazebo so it can be used successfully on python 3.x -- I don't expect many problems, except that protobuf 2.6 isn't quite py3 compatible yet (see https://github.com/google/protobuf/issues/7), but it looks like that will change very very soon. 

I'll try to link my PR's to this bug until all the issues are resolved.

**Comment by jpieper** [comment]
Thanks.  I had originally intended for it to be python 2 or 3 compatible, but when I couldn't easily get protobuf working back when I started, I gave up, and some of the python3 requirements may have bitrotted in that time.

**Comment by virtuald** [comment]
I've got a working protobuf working on my machine, but it was a pain. Hopefully that will be resolved soon. From what I've hit so far, there are byte/str problems -- but I'll work through those tonight and hopefully have it working. Then we just wait for protobuf.

**Comment by jpieper** [comment]
For the record, when you think you have it working, you should update the tox.ini to include at least one python3 variant and get all the tests to pass there as well.

**Comment by virtuald** [comment]
Yup yup. I'll also update the travis.yml too. I don't want to update that until protobuf has a pypi package that is python3 compatible.
